### ISSUE TITLE
Homepage: make external content banner focus- & clickable without mouse

### ISSUE THREAD

**Issue by FoxyHunter7** [eurofurence/website#32]
[WCAG: Keyboard (Level A) ](www.w3.org/WAI/WCAG21/Understanding/keyboard.html)
[WCAG: Name, Role, Value (Level A) ](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

The element is not focusable nor clickable with keyboard.
The element is not announced as a button or clickable item, only the visual UI hints at it (:hover).
